drivers/ph_oem/ph_oem.c

```diff
@@ -0,0 +1,440 @@
+/*
+ * Copyright (C) 2019 University of Applied Sciences Emden / Leer
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_ph_oem
+ * @{
+ *
+ * @file
+ * @brief       pH OEM device driver
+ *
+ * @author      Igor Knippenberg <igor.knippenberg@gmail.com>
+ * @}
+ */
+
+#include "xtimer.h"
+#include "assert.h"
+#include "periph/i2c.h"
+#include "periph/gpio.h"
+
+#include "ph_oem.h"
+#include "ph_oem_params.h"
+#include "ph_oem_regs.h"
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+#define I2C (dev->params.i2c)
+#define ADDR (dev->params.addr)
+#define IRQ_OPTION (dev->params.irq_option)
+
+static int ph_oem_init_test(const ph_oem_t *dev);
+
+int ph_oem_init(ph_oem_t *dev, const ph_oem_params_t *params)
+{
+    assert(dev && params);
+
+    dev->params = *params;
+
+    return ph_oem_init_test(dev);
+}
+
+static int ph_oem_init_test(const ph_oem_t *dev)
```

Comment:
Here and bellow, according to the [Coding conventions](https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions#Naming) private functions do not have the module prefix.

Reply:
Fixed the naming of the private functions